Automattic/pocket-casts-ios
# Tracks: Adds analytics to the Auto Add to Up Next settings

| 📘 Project: #154 | 🛫 Depends on: #310 |
|:---:|:---:|

- `settings_auto_add_up_next_shown`: When the Auto Add to Up next setting is displayed to the user
- `settings_auto_add_up_next_auto_add_limit_changed`: When the user changes the Auto Add Limit option
- `settings_auto_add_up_next_limit_reached_changed`: When the user changes the If Limit Reached option
- `settings_auto_add_up_next_podcasts_changed`: When the use changes which podcasts should be auto added to the up next
- `settings_auto_add_up_next_podcast_position_option_changed`: When the user changes a podcasts position option

## To test

1. Go to the profile tab
2. Tap on the cog
3. Tap on Auto Add to Up Next
4. ✅ `🔵 Tracked: settings_auto_add_up_next_shown`
9. Go through each setting and change its value
12. Check console and verify each setting has a corresponding event in console and the value displayed is the value that you selected
13. If the option changed is a switch then the event should be suffixed `_toggle` if it's a value change it should be suffixed with `_changed`


## Checklist
